src/test/mockFactory.ts

```diff
@@ -21,7 +21,8 @@ export const createTestServiceBuildAndUpdatePipelineYaml = (
   serviceName = "my-service",
   relativeServicePathFormatted = "./my-service",
   ringBranches: string[] = ["master", "qa", "test"],
-  variableGroups: string[] = []
+  variableGroups: string[] = [],
+  serviceBuildVariables: string[] = ["${serviceBuildVariables}"]
```

Comment:
This value looks a bit suspect. For the base test, shouldn't this also just be an empty array?